multimodal/src/autogluon/multimodal/predictor.py

```diff
@@ -1764,7 +1764,7 @@ def _on_predict_start(
         else:  # called .fit() or .load()
             df_preprocessor = self._df_preprocessor
 
-        data_processors = copy.deepcopy(self._data_processors)
+        data_processors = self._data_processors
```

Comment:
I notice there is a `data_processors.pop` operation right below this line, will this cause issues due to no longer performing a deepcopy?

Reply:
Yes, you're right. It will become an issue when we call predictor.fit() twice, as indicated in some of the test cases.

Reply:
We can also just do `copy.copy()`, which will be shallow and won't impact the fact that LABEL key will be dropped.

Reply:
Changed this to construct a new data_processors. Constructing a new data_processor is a lot (100x) faster than making a deepcopy. In my experiments, making a deepcopy would cost 40ms, while constructing a new data_processor would take less than 0.1ms.